src/categories-template.html

```diff
@@ -9,7 +9,7 @@ <h1 tabindex="-1">Choose an activity</h1>
     </li>
     {{/each}}
     <li>
-      <button data-action="set-cat" data-category="i-dont-know">I don&#x2019;t know</button>
+      <button data-action="set-cat" data-category="i-dont-know">Pick at random</button>
```

Comment:
ok but I might change the data category to 'pick-random-cat' 

Just for the shear fun of it.